### ISSUE TITLE
Make it possible to build only the inbox portion of libs

### ISSUE THREAD

**Issue by MichalStrehovsky** [dotnet/runtime#51168]
`build.cmd libs` will build all libraries, but most people working in this repo only want what's inbox in .NET.

~~I have a:~~

```xml
    <ProjectExclusions Include="$(MSBuildThisFileDirectory)Microsoft.Extensions.*\src\*.csproj" />
```

~~line in my libraries\src.proj and this speeds up the libraries build by 15%, but it would be nice to be able to easily exclude the rest of the useless libraries and get additional boost.~~

EDIT: I just realized the above is a cargo cult I've been doing and apparently doesn't work (anymore?).

**Comment by MichalStrehovsky** [comment]
Duping this to #65492.

**Comment by ViktorHofer** [comment]
Sorry @MichalStrehovsky I forgot about this issue. Thanks for duping it against the one that I just opened. This will be implemented with https://github.com/dotnet/runtime/pull/64000, then you can just build the sfx by invoking `build.cmd/sh libs.sfx`.

**Comment by MichalStrehovsky** [comment]
> Sorry @MichalStrehovsky I forgot about this issue. Thanks for duping it against the one that I just opened. This will be implemented with #64000, then you can just build the sfx by invoking `build.cmd/sh libs.sfx`.

I don't mind there's a new issue with more detail, and doubly so because there's also a pull request to go along with it. Opening issues is easy, pull requests are hard. Thank you for working on this! I owe you a drink!
